README.md

```diff
@@ -318,6 +318,9 @@ instructions below may not work properly on Windows.
 
 * If you want to make an executable, simply run `./scripts/build`. The binary will appear in the `./out` folder.
 
+## Security
```

Comment:
i'm fine with that text.  do you want to modify it yourself?  and just close this pull?